internal/app/referenceserver/checks.go

```diff
@@ -39,10 +42,29 @@ const (
 	connectStreamContentTypePrefix = "application/connect+"
 	connectContentTypePrefix       = connectUnaryContentTypePrefix
 
+	connectTimeoutHeader = "connect-timeout-ms"
+	grpcTimeoutHeader    = "grpc-timeout"
+
 	codecProto = "proto"
 	codecJSON  = "json"
 )
 
+type int32Enum interface {
+	~int32
+	protoreflect.Enum
+}
+
+type feedbackPrinter struct {
+	p            internal.Printer
+	testCaseName string
+}
+
+func (p *feedbackPrinter) Printf(format string, args ...any) {
+	p.p.PrefixPrintf(p.testCaseName, format, args...)
+}
```

Comment:
I moved these up to conform to our style guide, which wants types (and their methods) to appear above functions.